benchmarks/models/fs_bart.sh

```diff
@@ -22,6 +22,12 @@ source utils.sh
     valid \
     32/64/128/256 \
     --max-tokens 131072
+./benchmark.sh \
+    fairseq+fastseq+el \
+    bart.large.cnn \
+    cnn_dm/len-1024.bin \
+    valid \
+    320
```

Comment:
Try a few more batch_size, e.g., 128/256/320?

Reply:
EL attn basically helps in saving memory to that bs can be increased, think of it like cache optimization. So comparing same batch sizes is not really useful..